Makefile

```diff
@@ -35,6 +35,7 @@ install-deps:
 	sudo mkdir -p /opt/wasi-sdk
 	sudo mv wasi-sdk-15.0/* /opt/wasi-sdk/
 	sudo rm -rf wasi-sdk-*
+	brew install redis
```

Comment:
Is this a reliable installation of redis for Ubuntu? I know `brew` works just fine, just wondering if the tap is reliably updated.

Reply:
could you elaboate on your concerns? What problems would `brew` bring?

Reply:
It could be serving an old version of redis because the brew tap isn't updated.

Reply:
Yeah~ My concern was mostly that the linux tap and the macos one might have diff versions, but, if your experience has been ok, I'm good to use brew as a package manager for our Ubuntu tests too!

Reply:
I think it will be okay for now~